### PR TITLE
Karma Quest: Create Demand and show demand list

### PR DESCRIPTION
**Pull request by sohailamjad12**
1-Request List table Created 
2-Create demand Ui with integration
3-Copy demand Ui with integration
4-view Demand Details

### CODE REVIEW

File: project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <div class="full-table-container">
+        <form [formGroup]="requestForm">
+            <mat-radio-group formControlName="assignee">
+                <table mat-table [dataSource]="providers" class="mat-elevation-z8 full-height">
+    
+                    <!-- Radio Button Column -->
+                    <ng-container matColumnDef="select">
+                        <th mat-header-cell *matHeaderCellDef></th>
+                        <td mat-cell *matCellDef="let provider">
+                            <mat-radio-button [value]="provider.orgName"></mat-radio-button>
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.html:12]: please do optional checks wherever its needed in all files

File: project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@Component({
+  selector: 'ws-app-assign-list-popup',
+  templateUrl: './assign-list-popup.component.html',
+  styleUrls: ['./assign-list-popup.component.scss'],
+})
+export class AssignListPopupComponent implements OnInit {
+  requestForm!: FormGroup
+  providers = [
+    { orgName: 'Microsoft',
+     details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec suscipit orci in ultricies aliq.',
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.ts:13]: is dummy data required?

File: project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+      </div>
+    </mat-card-actions>
+  </mat-card>
+  
+  <mat-card class="competency-view-card" *ngIf="dData && !dData.name">
+    <mat-card-content class="flex flex-1 custom margin-remove">
+      <div class="flex">
+        <h3 class="mat-h3">Competency area: &nbsp;</h3>
+        <span class="mat-h3 font-medium">{{dData.competencyArea}}</span>
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.html:39]: same as above , do optional check

File: project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  }
+  remove() {
+    this.dialogRef.close({
+      id: this.dData.id || this.dData,
+      action: 'DELETE',
+    })
+  }
+
+  formate(text: string): SafeHtml {
+    let newText = '<ul class="pl-6">'
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.ts:45]: why we are doing like this?

File: project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.html (hunk 1 above)
Comment (sohailamjad12) [reply project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.html:12]: api is not provided yet..we need to change it again once we get the api

File: project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.ts (hunk 2 above)
Comment (sohailamjad12) [reply project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.ts:13]: api is not provided yet ..will mention with qa people

File: project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.ts (hunk 4 above)
Comment (sohailamjad12) [reply project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.ts:45]: This features already implemented by Nandini..i have just copied here

File: project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.html (hunk 3 above)
Comment (sohailamjad12) [reply project/ws/app/src/lib/routes/home/components/request-list/competency-view/competency-view.component.html:39]: added optional checks

**Pull request merged**